frontend/src/App.js

```diff
@@ -50,6 +52,16 @@ function App() {
             />
           </>
         )}
+        {(hasRole(currentUser, "ROLE_USER") ||
+          hasRole(currentUser, "ROLE_INSTRUCTOR")) && (
```

Comment:
I think only ROLE_ADMIN and ROLE_INSTRUCTOR should have access to this, not end users.  End users should use the "Other" field if their request type isn't in the table.